addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3808,8 +3808,9 @@ export class OdooEditor extends EventTarget {
             }
             // Find previous character.
             let previousCharacter = focusOffset > 0 && focusNode.textContent[focusOffset - 1];
-            if (!previousCharacter) {
-                focusNode = previousLeaf(focusNode);
+            const previousNode = previousLeaf(focusNode);
```

Comment:
This is going to conflict with [that](https://github.com/odoo/odoo/pull/154217/files#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR3812) so I suggest we wait as it's getting merged so you can resolve the conflicts and make sure that fix is not lost.